Source/Urho3D/Graphics/OpenGL/OGLGraphics.cpp

```diff
@@ -785,7 +797,7 @@ void Graphics::Draw(PrimitiveType type, unsigned indexStart, unsigned indexCount
 void Graphics::DrawInstanced(PrimitiveType type, unsigned indexStart, unsigned indexCount, unsigned minVertex, unsigned vertexCount,
     unsigned instanceCount)
 {
-#ifndef GL_ES_VERSION_2_0
+#if !defined(GL_ES_VERSION_2_0) || defined(__EMSCRIPTEN__)

```

Comment:
Note: Before this pull request, I found it odd that the function `Graphics::DrawInstanced()` existed even when `GL_ES_VERSION_2_0` is defined, but it compiled down to an empty function body. That could possibly lead to silent failure behavior if some code accidentally calls the no-op `Graphics::DrawInstanced`? It might be more desirable to completely `#ifdef` out the whole `Graphics::DrawInstanced()` function from existing in the first place when instancing is known to be not supported at compile-time. This is probably a non-concern when/if instancing support is added to GLES2 in the future.